### PR TITLE
Added Google OAuth in order to create an event, Fixes #13

### LINKED ISSUE #13: As a user, I want to be able to use my google account

### PR DESCRIPTION
**Pull request by ClementGautier**

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -16,9 +16,10 @@ public function registerBundles()
             new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
             new Doctrine\Bundle\MongoDBBundle\DoctrineMongoDBBundle(),
             new FOS\RestBundle\FOSRestBundle(),
+            new Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle(),
+            new HWI\Bundle\OAuthBundle\HWIOAuthBundle(),
             new Akeneo\Bundle\ApiBundle\AkeneoApiBundle(),
             new Akeneo\Bundle\FrontofficeBundle\AkeneoFrontofficeBundle(),
```
Comment (jmleroux) [inline app/AppKernel.php]: we should order the `use` statements

File: app/config/config.yml [hunk 2]
```diff
@@ -82,3 +82,12 @@ fos_rest:
...
     exception:
         enabled: true
+
+hwi_oauth:
+    firewall_name: main
+    resource_owners:
+        google:
+            type:                google
+            client_id:           262100838903-rp0ithf8hh62nh3tef51vnquqirohdkg.apps.googleusercontent.com
+            client_secret:       Qb14mfzbx8iCHyLP6uC2jRVy
```
Comment (jmleroux) [inline app/config/config.yml]: these values should be parameters

File: composer.json [hunk 3]
```diff
@@ -19,7 +19,8 @@
         "friendsofsymfony/rest-bundle": "^1.7",
         "doctrine/doctrine-fixtures-bundle": "^2.2",
         "guzzlehttp/guzzle": "~6.0",
-        "symfony/swiftmailer-bundle": "^2.3"
+        "symfony/swiftmailer-bundle": "^2.3",
+        "hwi/oauth-bundle": "^0.3.9"
```
Comment (jmleroux) [inline composer.json]: would be better if sorted by package name

File: spec/Akeneo/Bundle/FrontofficeBundle/Form/EventSubscriber/EventToApiSubscriberSpec.php [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+    function it_submits_null_values(FormEvent $event)
+    {
+        $event->getData()->willReturn(null);
+        $event->setData(Argument::any())->shouldNotBeCalled();
+
+        $this->submit($event);
+    }
+
+    function it_submits_form_values($tokenStorage, TokenInterface $token, OAuthUser $user, FormEvent $event)
```
Comment (jmleroux) [inline spec/Akeneo/Bundle/FrontofficeBundle/Form/EventSubscriber/EventToApiSubscriberSpec.php]: typehinted first

File: src/Akeneo/Bundle/FrontofficeBundle/Form/Type/AddEventType.php [hunk 5]
```diff
@@ -19,6 +18,25 @@
 class AddEventType extends AbstractType
 {
     /**
+     * @var EventSubscriberInterface
```
Comment (jmleroux) [inline src/Akeneo/Bundle/FrontofficeBundle/Form/Type/AddEventType.php]: Akeneo use one line for `@var` statements

File: src/Akeneo/Bundle/FrontofficeBundle/Resources/public/js/map.js [hunk 6]
```diff
@@ -64,7 +64,7 @@ function deleteMarkers() {
 }
 
 function generateUrl(from) {
-    var url = '/app_dev.php/api/events';
+    var url = '/api/events';
```
Comment (jmleroux) [inline src/Akeneo/Bundle/FrontofficeBundle/Resources/public/js/map.js]: do we plan to use FOSRouting ?

File: app/AppKernel.php (hunk 1 above)
Comment (ClementGautier) [inline app/AppKernel.php]: its not a use statement

File: app/config/config.yml (hunk 2 above)
Comment (ClementGautier) [inline app/config/config.yml]: agree

File: spec/Akeneo/Bundle/FrontofficeBundle/Form/EventSubscriber/EventToApiSubscriberSpec.php (hunk 4 above)
Comment (ClementGautier) [inline spec/Akeneo/Bundle/FrontofficeBundle/Form/EventSubscriber/EventToApiSubscriberSpec.php]: not sure that I can do that, its a phpspec file

File: src/Akeneo/Bundle/FrontofficeBundle/Resources/public/js/map.js (hunk 6 above)
Comment (ClementGautier) [inline src/Akeneo/Bundle/FrontofficeBundle/Resources/public/js/map.js]: we could yes

File: spec/Akeneo/Bundle/FrontofficeBundle/Form/EventSubscriber/EventToApiSubscriberSpec.php (hunk 4 above)
Comment (willy-ahva) [inline spec/Akeneo/Bundle/FrontofficeBundle/Form/EventSubscriber/EventToApiSubscriberSpec.php]: You shouldn't do that ;)

Comment (jmleroux) [inline spec/Akeneo/Bundle/FrontofficeBundle/Form/EventSubscriber/EventToApiSubscriberSpec.php]: @willy-ahva why ?

Comment (ClementGautier) [inline spec/Akeneo/Bundle/FrontofficeBundle/Form/EventSubscriber/EventToApiSubscriberSpec.php]: actually it work, phpspec is full of magic ^^

**Pull request merged**